EAGLE Libraries/HyTechDevices.lbr

```diff
@@ -8764,6 +8862,22 @@ Leakage Inductance: 0.50 uH max
 </technology>
 </technologies>
 </device>
+<device name="MB" package="MINI-BLADE_FUSE_HOLDER">
+<connects>
+<connect gate="G$1" pin="1" pad="L1 L2"/>
+<connect gate="G$1" pin="2" pad="R1 R2"/>
+</connects>
+<technologies>
+<technology name="10">
+<attribute name="CURRENT" value="10A"/>
+<attribute name="DKPN" value="F991-ND"/>
+<attribute name="MANUFACTURER" value="Littelfuse Inc."/>
+<attribute name="MOPN" value="576-0297010.WXNV"/>
+<attribute name="MPN" value="0297010.WXNV"/>
+<attribute name="VOLTAGE" value="32V"/>
+</technology>
+</technologies>
+</device>
```

Comment:
fuses are not fuse holders. delete this and include the fuse holder part from main with a 10A external fuse
```suggestion
```